packages/ignite-dev-screens/templates/ButtonBox.js

```diff
@@ -1,17 +1,14 @@
-// @flow
 import React from 'react'
 import { TouchableOpacity, Text, Image } from 'react-native'
 import styles from './Styles/ButtonBoxStyles'
 
-type ButtonBoxProps = {
-  onPress: () => void,
-  image?: string,
-  style?: Object,
-  text?: string
-};
-
 export default class ButtonBox extends React.Component {
-  props: ButtonBoxProps
+  props = {
```

Comment:
For props, let's convert these to `static propTypes=` definitions.  If you really want, you can create a `static defaultProps` as well, but I find we rarely do this in practice.